src/lib.rs

```diff
@@ -47,6 +47,7 @@
 //!
 #![deny(missing_debug_implementations)]
 #![warn(missing_docs, unused_extern_crates, trivial_casts, unused_results)]
+#![recursion_limit = "128"]
```

Comment:
Can you please tell me why we need a recursion limit? I don't remember any use of recursion in this lib.

Reply:
This is the compiler's recursive operation limit. Recursion is used internally by the `quick_error!` macro. Once a significant number of variants are declared, which is the case here, we need to push this value a bit higher.